templates/main/13_all.go.tpl

```diff
@@ -13,6 +13,9 @@ func {{$alias.UpPlural}}(mods ...qm.QueryMod) {{$alias.DownSingular}}Query {
     if len(queries.GetSelect(q)) == 0 {
         queries.SetSelect(q, []string{"{{$schemaTable}}.*"})
     }
+    {{if and .AddSoftDeletes $canSoftDelete .AutoColumns.Deleted -}}
+        queries.SetRemoveSoftDeleteRgx(removeSoftDeleteRgx)
+    {{end}}
```

Comment:
This should be in a singleton template. It only needs to be called once. here it will be called for every table.

Reply:
It makes sense, changed!